docs/kb/release-notes.md

```diff
@@ -8,6 +8,11 @@ This page contains the release notes for the Pardot API and related documentatio
 * updated "Reading Emails" section to correctly label the required param `email_id` instead of `email`.
 * updated "Reading Emails" section to correctly label the required param `list_email_id` instead of `email`.
 
+### Prospects
+* batchCreate endpoint now returns an attributes node for JSON responses.
```

Comment:
I think it would be helpful to mention something along the lines of "to know if the request was successful". Thoughts?